shell/completion.zsh

```diff
@@ -308,7 +308,7 @@ fzf-completion() {
   setopt localoptions noshwordsplit noksh_arrays noposixbuiltins
 
   # Check if at least one completion system (old or new) is active
-  if ! zmodload -F zsh/parameter p:functions 2>/dev/null || ! (( $+functions[compdef] )); then
+  if ! zmodload -F zsh/parameter p:functions 2>/dev/null || ! (( ${+functions[compdef]} )); then
```

Comment:
The `{}` (curly brackets) aren't needed since we already disable `ksh_arrays` in the local options. We should also disable it in the `fzf-history-widget` to avoid `ksh_arrays` issues in future commits for these widgets.

Reply:
You mean curlies here can be omitted? If it wouldn't hurt to have them here, I'd suggest keeping them for consistency.

Reply:
> You mean curlies here can be omitted?

For this specific widget, they could because of `noksh_arrays`, but it wouldn't hurt to have them.